vocab-importer/app.py

```
@@ -0,0 +1,530 @@
+    # Generator panel
+    st.markdown('<div class="panel">', unsafe_allow_html=True)
+    st.markdown("### ✨ Generate Vocabulary")
+
+    # Get available Ollama models
+    def get_ollama_models():
+        try:
+            response = requests.get("http://localhost:11434/api/tags")
+            if response.status_code == 200:
+                models = response.json().get("models", [])
+                return [f"ollama:{model['name']}" for model in models]
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)